### PR TITLE
Added auto disenchanter

### PR DESCRIPTION
**Pull request by Phoenix-Starlight**
I am aware that this is a very short, not detailed page about the auto disenchanter, but it sums up the process of how it works. I also fixed exactly 1 typo on Slimefun in a nutshell.

### CODE REVIEW

File: pages/Auto-Disenchanter.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Auto Disenchanter
+It requires a source of electricity, see [Electric Machines](https://github.com/Slimefun/Slimefun4/wiki/Electric-Machines).
+You place the slimefun or vanilla tool in the left slot, and a indicator will appear showing the progress and how much energy it has. Upon completion, the right side will produce the respective tool, unenchanted, and a enchanted book holding the enchants of that tool.
```
Comment (TheBusyBiscuit) [inline pages/Auto-Disenchanter.md:3]: This description is wrong in many ways.
You also need a book for example, not just putting in a tool.

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]

File: pages/Auto-Disenchanter.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Auto Disenchanter
+It requires a source of electricity, see [Electric Machines](https://github.com/Slimefun/Slimefun4/wiki/Electric-Machines).
+You place the slimefun or vanilla tool in the left slot along with a book, and a indicator will appear showing the progress and how much energy it has. Upon completion, the right side will produce the respective tool, unenchanted, and a enchanted book holding the enchants of that tool.
```
Comment (svr333) [inline pages/Auto-Disenchanter.md:3]: an indicator*
showing the progress on how much*
the respective tool, disenchanted* (technically unenchanted is correct but I feel this is more accurate)
an enchanted book*

**Review by svr333 — APPROVED** [review]

File: pages/Auto-Disenchanter.md (hunk 2 above)
Comment (svr333) [inline pages/Auto-Disenchanter.md:3]: an indicator*
showing the progress on how much*
the respective tool, disenchanted* (technically unenchanted is correct but I feel this is more accurate)
an enchanted book*

**Review by svr333 — CHANGES_REQUESTED** [review]
Accidentally approved earlier

File: pages/Auto-Disenchanter.md [hunk 3]
```diff
@@ -1,3 +1,3 @@
 # Auto Disenchanter
 It requires a source of electricity, see [Electric Machines](https://github.com/Slimefun/Slimefun4/wiki/Electric-Machines).
-You place the slimefun or vanilla tool in the left slot along with a book, and a indicator will appear showing the progress and how much energy it has. Upon completion, the right side will produce the respective tool, unenchanted, and a enchanted book holding the enchants of that tool.
+You place the slimefun or vanilla tool in the left slot along with a book, and a indicator will appear showing the progress and how much energy it has. Upon completion, the right side will produce the respective tool, unenchanted, and an enchanted book holding the enchants of that tool.
```
Comment (svr333) [inline pages/Auto-Disenchanter.md:3]: You only fixed 1 out of 4 mentioned mistakes:
an indicator*
showing the progress on how much*
the respective tool, disenchanted* (technically unenchanted is correct but I feel this is more accurate)

**Review by svr333 — CHANGES_REQUESTED** [review]

**Comment by Phoenix-Starlight** [comment]
Noted.

**Comment by svr333** [comment]
Didnt know you could reply via email, cool

**Comment by Phoenix-Starlight** [comment]
It was shown in the email notification, so I said I noted it and would
amend it when I can get a hold of a desktop platform.

**Comment by Phoenix-Starlight** [comment]
Quick bump here.

**Review by svr333 — APPROVED** [review]

File: pages/Auto-Disenchanter.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# Auto Disenchanter
```
Comment (TheBusyBiscuit) [inline pages/Auto-Disenchanter.md:1]: Articles may not start with a heading, please refer to this guide for more help:
https://github.com/Slimefun/Slimefun4/wiki/Expanding-the-Wiki#step-3-general-guidelines-dos-and-donts

File: pages/Auto-Disenchanter.md [hunk 5]
```diff
@@ -0,0 +1,3 @@
...
+It requires a source of electricity, see [Electric Machines](https://github.com/Slimefun/Slimefun4/wiki/Electric-Machines).
+You place the slimefun or vanilla tool in the left slot along with a book, and an indicator will appear showing the progress on the process. Upon completion, the right side will produce the respective tool, disenchanted, and an enchanted book holding the enchants of that tool.
```
Comment (TheBusyBiscuit) [inline pages/Auto-Disenchanter.md:3]: There should be a `## Usage` heading before this sentence.

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]

File: pages/Auto-Disenchanter.md (hunk 5 above)
Comment (Phoenix-Starlight) [reply pages/Auto-Disenchanter.md:4]: Edit performed, synced the repo so that I don't roll anything back by accident.

**Review by svr333 — APPROVED** [review]

**Review by Phoenix-Starlight — COMMENTED** [review]
Changes addressed.

**Review by TheBusyBiscuit — APPROVED** [review]
Very basic but okay.

**Pull request merged**